ci-operator/config/openshift-pipelines/performance/tools/manage-perf-crons.py

```
@@ -0,0 +1,731 @@
+        jobs = families[fkey]
+        print(f"\n{label} ({len(jobs)} cron jobs)")
+        print(header)
+        print(sep)
+        for job in jobs:
+            ver = get_version(job) or "?"
+            var = get_variant(job)
+            cron = parse_cron(job["cron"])
+            dom = "daily" if cron["dom"] == "*" and cron["dow"] == "*" else cron["dom"]
+            print(f"  {ver:<10} {var:<16} h{cron['hour']:<4} {dom}")
+
+    print(f"\nVARIANT-DAY SUMMARY")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)